### ISSUE TITLE
agw_health_cli fails with message "Error: 'logLevel'"

### ISSUE THREAD

**Issue by jordanvrtanoski** [magma/magma#6379]
### Your Environment

- 1.5, 1.6
- Access Gateway
- AGW cli tools
- bare-metal (AGW), VM (AGW)
- architecture: X86, ARM64
- Debian 9, Ubuntu 20.04

### Describe the Issue

Executing the "agw_heath_cli.py" stops with message "Error: 'logLeve'" 

**To Reproduce**

```
root@magma-001:~/.ssh# agw_health_cli.py
Access Gateway health summary

Using SubscriberDB
#eNBs connected:    0 	 (run `enodebd_cli.py get_all_status` for more details)
#IPs allocated:     0 	 (run `mobility_cli.py list_allocated_ips` for more details)
#UEs connected:     0 	 (run `mobility_cli.py get_subscriber_table` for more details)
#Core dumps:        0 	 (run `ls /var/core` to see core dumps)
Earliest core-dump: -, Latest core-dump: None
Registration success rate: No Attach requests (0 requests, 0 accepted)

/usr/local/lib/python3.5/dist-packages/magma/common/cert_utils.py:16: CryptographyDeprecationWarning: Python 3.5 support will be dropped in the next release of cryptography. Please upgrade your Python.
  from cryptography import x509
Health Summary

Gateway <-> Controller connectivity
1. -- Testing TCP connection to controller.magma.test:7443 --
2. -- Testing Certificate --
3. -- Testing SSL --
4. -- Creating direct cloud checkin --
5. -- Creating proxy cloud checkin --
Success!

Error: 'logLevel'
```

**Expected behavior**
In older version, the CLI was presenting the list of services and their status.

**Screenshots**
see above

**Additional context**
The problem is observed since v1.5 and is reproducible on virtual environments as well as on bare metal.

**Comment by jordanvrtanoski** [comment]
Additional information:
Error appears in `get_error_summary()` in `/root/magma/orc8r/gateway/python/magma/common/health/health_service.py`

**Comment by jordanvrtanoski** [comment]
Fixed with #6381, pending merge
